README.md

```diff
@@ -1,130 +1,61 @@
-- Log in to npm:
-  - npm login
-  - Credentials:
-  - username: osmobot-devops
-  - password: Get this from someone on the software team.
-  - e-mail: devops@osmobot.com
--- parses hex string using JS parsing library
--- has a lot of functionality re: hex string parsing, retrieving specific data from hex string
-```
-- Joining a data collection log with node data
--- Purpose
---- During an experiment, you often have a metadata "log" of times when you were collecting a specific type of data using a node.
---- This joiner allows you to augment your node data frame with this data. For each node data row, it finds the associated metadata and adds it in as additional columns. If there is no corresponding metadata row for a node data row, the node data row will get data_set_id of -1 or NaN and can easily be dropped from the resultant DataFrame.
-- Node Start and Stopper Neptune (candidate for sdk)
--- UI for ssh'ing into a hub and starting stoping the hub.service
-
-- Technician Access to Calculation Details-Aggressive anti-caching
--- Pulling calculation details from the DB
---- The calculation details table has all kinds of handy internal intermediate details for each calculation.
---- Getting the data from the DB as part of your notebook means you can get updated data just by re-running the notebook. However, since the database table is subject to change, it's also smart to save off the data in a file alongside the notebook so that you can run the notebook later without requiring database access.
---- Recommended practice is to use the database code while the experiment is in progress or when doing a single data pull, but always save that data off to a CSV and switch to using the CSV before you archive the notebook.
-
-- Technician Access to Calculation Details
--- Pulling calculation details from the DB
---- The calculation details table has all kinds of handy internal intermediate details for each calculation.
---- Getting the data from the DB as part of your notebook means you can get updated data just by re-running the notebook. However, since the database table is subject to change, it's also smart to save off the data in a file alongside the notebook so that you can run the notebook later without requiring database access.
---- Recommended practice is to use the database code while the experiment is in progress or when doing a single data pull, but always save that data off to a CSV and switch to using the CSV before you archive the notebook.
-
-# End Jason Notebook
+# Jupyter Notebook Tools
+This is a set of tools that is designed to facilitate science in the context of Osmo Systems.
+The code herein generally expects to be run in the context of a jupyter notebook. TODO (PR feedback plz): eventually it might be used elsewhere. is it OK to put it all here for now and potentially have to refactor/move it later?
```

Comment:
This seems most appropriate as the location for now as our usage patterns have not emerged outside of the need of Jupyter notebook experimentation.

Reply:
roger.